extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -93,7 +104,61 @@ class InstrumentationService {
           }
         });
       }
+    } else if (eventType === AnalyticsEventType.CLICK) {
+      this.ensureClickListener_();
+      this.clickObservable_.add(
+          this.createSelectiveListener_(listener, opt_selector));
```

Comment:
What would an empty/undefined/null `opt_selector` mean in this case? Should there be some check to require it or issue a warning?